nats/js/manager.py

```diff
@@ -68,12 +68,13 @@ async def find_stream_name_by_subject(self, subject: str) -> str:
             raise NotFoundError
         return info['streams'][0]
 
-    async def stream_info(self, name: str) -> api.StreamInfo:
+    async def stream_info(self, name: str, subjects_filter: str = '>') -> api.StreamInfo:
         """
         Get the latest StreamInfo by stream name.
         """
+        req_data = json.dumps({"subjects_filter": subjects_filter})
         resp = await self._api_request(
-            f"{self._prefix}.STREAM.INFO.{name}", timeout=self._timeout
+            f"{self._prefix}.STREAM.INFO.{name}", req_data.encode(), timeout=self._timeout
```

Comment:
would prefer to send the subjects_filter JSON only when explicitly specifying a subject.